CHANGELOG.md

```diff
@@ -35,10 +35,45 @@ Markdown Spec](https://github.github.com/gfm/).
 ### Added
 
 ### Changed
+- Upgraded to Solr 8.3 - this includes our docker-compose setup as well as the
+  way Solr is configured
 
 ### Removed
 
 ### Migration
+- If you're using docker, rebuild your ONI image: `docker-compose build`
```

Comment:
I totally missed this step (my fault) because I saw the line below "docker developers" and jumped down to that without seeing this first command. I don't know if there's a different way to organize these or if we just want to chalk that up to the fault of the user, but just fyi!

Reply:
Also, I was getting all manner of 404 errors from ubuntu urls when trying to run this command until I found this in stackoverflow and added `--no-cache` to the end of the build command:
https://stackoverflow.com/a/37727984/4154134

I recommend that we either include that in these instructions or look into the second option listed in the answer

Reply:
Second option is probably a good idea.  I'll file a ticket.

Reply:
Filed ticket #454 for this